### PR TITLE
fixed tests for profile routes

### PR DESCRIPTION
**Pull request by RichardDominik**

### CODE REVIEW

File: tests/Feature/Profile/DefaultProfileGeneratorTest.php [hunk 1]
```diff
@@ -62,12 +62,10 @@ class ProfileController extends Controller
 /* Auto-generated admin routes */
 Route::middleware([\'auth:\' . config(\'admin-auth.defaults.guard\'), \'admin\'])->group(static function () {
     Route::prefix(\'admin\')->namespace(\'Admin\')->name(\'admin/\')->group(static function() {
-        Route::prefix(\'admin-users\')->name(\'admin-users/\')->group(static function() {
```
Comment (palypster) [inline tests/Feature/Profile/DefaultProfileGeneratorTest.php]: there was prefix `admin-users` before? Weird, I didn't know.

Comment (dejwCake) [reply tests/Feature/Profile/DefaultProfileGeneratorTest.php]: I think @RichardDominik introduced it few commits back.

File: tests/TestCase.php [hunk 2]
```diff
@@ -48,9 +48,9 @@ protected function getEnvironmentSetUp($app)
                 'driver' => 'pgsql',
                 'host' => 'testing',
                 'port' => '5432',
-                'database' => env('DB_DATABASE'),
-                'username' => env('DB_USERNAME'),
-                'password' => env('DB_PASSWORD'),
+                'database' => env('DB_DATABASE', 'laravel'),
```
Comment (dejwCake) [inline tests/TestCase.php]: Why do you need to provide default value here?, should not be set in phpunit file?

Comment (palypster) [reply tests/TestCase.php]: Yes, that's the second option. But no prob for me with default value too.

**Pull request merged**